app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>t("Sample App")</title>
```

Comment:
please apply Rails I18n in this project
refs: https://guides.rubyonrails.org/i18n.html#setup-the-rails-application-for-internationalization